kernel/src/utilities/helpers.rs

```diff
@@ -62,8 +62,8 @@ macro_rules! create_capability {
 #[macro_export]
 macro_rules! count_expressions {
     () => (0usize);
-    ($head:expr $(,)?) => (1usize);
-    ($head:expr, $($tail:expr),* $(,)?) => (1usize + count_expressions!($($tail),*));
+    ($head:expr_2021 $(,)?) => (1usize);
+    ($head:expr_2021, $($tail:expr_2021),* $(,)?) => (1usize + count_expressions!($($tail),*));
```

Comment:
Seems like we just want to use `expr`.

https://doc.rust-lang.org/edition-guide/rust-2024/macro-fragment-specifiers.html